packages/components/scripts/generateDSRComponents.ts

```diff
@@ -49,7 +49,7 @@ const generateDSRComponents = (): void => {
         .replace(/\n  public componentDidRender\(\): void {[\s\S]+?\n  }\n/g, '')
         .replace(/\n  public disconnectedCallback\(\): void {[\s\S]+?\n  }\n/g, '')
         .replace(/\n  public componentShouldUpdate\([\s\S]+?\n  }\n/g, '')
-        .replace(/\n  private (?!get).*{[\s\S]+?\n  };?\n/g, '') // private methods without getters
+        .replace(/\n  private (?!get).*\([\s\S]+?\n  };?\n/g, '') // private methods without getters
```

Comment:
What do we need this change for? Because now the regex is somewhat unbalanced when in the past it matched stuff between `{` and `}` but now it is between `(` and `}`.

Reply:
We did not have the multiline case before 😕 ![image](https://github.com/porsche-design-system/porsche-design-system/assets/109523972/bf3b5c91-6756-4f3f-8c2d-46ec85c17cf2) 
![image](https://github.com/porsche-design-system/porsche-design-system/assets/109523972/2dd6e287-39e7-4c0b-be6b-e42f09db35a1)

Reply:
But then let's try to make the regex balanced never the less.